src/ziggurat/mapper.clj

```diff
@@ -16,42 +16,38 @@
 
 (defn mapper-func [mapper-fn channels]
   (fn [message-payload]
-    (let [service-name               (:app-name (ziggurat-config))
-          topic-entity               (:topic-entity message-payload)
+    (let [topic-entity               (:topic-entity message-payload)
           topic-entity-name          (name topic-entity)
           new-relic-transaction-name (str topic-entity-name ".handler-fn")
           default-namespace          "message-processing"
-          metric-namespaces          [service-name topic-entity-name default-namespace]
           additional-tags            {:topic_name topic-entity-name}
           default-namespaces         [default-namespace]
```

Comment:
Do we still need a vector?

Reply:
I changed the behavior to pass a string instead... done